server/routes/events.ts

```diff
@@ -22,8 +22,15 @@ import {
   SpecialEvent,
   ProcessedSpecialEvent,
 } from "../types";
+//Logger
+import { logger } from "../loggerUtils/logger";
+import dotenv from "dotenv";
 
 const router = express.Router();
+dotenv.config();
+
+const TODAY =
+  process.env.NODE_ENV !== "dev" ? "TODAY()" : `"${process.env.TODAY}"`;
```

Comment:
Same thing here.

Reply:
Yeah, I'll make that change. I didn't intend to make this part of the same PR, but with that change this will resolve #55 in addition to #45.

Reply:
Yeah, I was going to say i'd prefer these to be separate PRs since they're totally unrelated. Not a big deal here, but keep in mind for future.